docs/vaahflutter/basics/getting-started.md

```diff
@@ -9,22 +9,34 @@
 
 ## How to run the app
 
-### Step 1. Clone repository
+### Step 1. Clone Repository
 
 To clone the repository, you'll need to run the below command in your terminal. Make sure [git](https://git-scm.com/downloads) is installed in your system.
 
 ```bash
 git clone ...
```

Comment:
@we-prajapati-c001 write this url to download the zip file `https://github.com/webreinvent/vaahflutter/archive/refs/heads/master.zip`

Reply:
yes sir